src/containers/Tenant/QueryEditor/SaveQuery/SaveQuery.scss

```diff
@@ -1,17 +1,19 @@
-@import '../../../../styles/mixins.scss';
-
 .kv-save-query {
-    &__dialog-body {
+    &__dialog-row {
         display: flex;
         align-items: flex-start;
+
+        & + & {
+            margin-top: var(--yc-text-body-line-height);
+        }
     }
     &__field-title {
         margin-right: 12px;
 
         font-weight: 500;
-        line-height: 32px;
+        line-height: 28px;
```

Comment:
the height of the text field
this artificially aligns the label with the field by baseline
align-items: baseline for the parent doesn’t work without extra refactoring